Blog: PostHog vs Hotjar

Lior539 opened the pull request:
## Changes

Added comparison

## Checklist
- [x] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [x] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [x] Words are spelled using American English
- [x] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)
- [x] If I moved a page, I added a redirect in `vercel.json`

Lior539 commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,301 @@
+
+In this guide, we'll cover:
+
+- Hotjar and PostHog [feature comparison](#feature-comparison)
+- [Session replay](#session-replay) capabilities
+- [Privacy](#privacy-compliance) and [security](#admin--security)
+- [Frequently asked questions](#frequently-asked-questions)
```
I wanted to check these work correctly, but kept getting a 404 when trying to visit this page locally. Anyone know how what may be causing this?

joethreepwood replied:
Actually, that's not what's causing the 404 - unsure about that - but these links still need to be corrected.

ivanagas replied:
You might need to run `gatsby clean` and `yarn` again to get rid of cached stuff.

For creating the heading links, I copy them from the links after running locally.

joethreepwood replied:
I figured out the 404 - it's a typo in a markdown table way below. <atr/> needed to be replaced with <tr/>, IIRC

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,301 @@
+
+### 3. PostHog is open source
+
+Because we build for engineers first, we're also open source. You can check out [PostHog’s source code](https://github.com/PostHog), [build integrations](/docs/apps/build) or [other services](/blog/how-we-built-an-app-server) on top of the product, and even [give feedback or interact with the team via GitHub](https://github.com/PostHog). Transparency, including transparent pricing, is one of our  ...
```
Best to switch the feedback link out to https://app.posthog.com/home#supportModal as that's the preferred feedback form now

Lior539 replied:
This link is only available for users that have already created a PostHog app

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,301 @@
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Capture user feedback with surveys. </td>
+        </tr>
+        <atr>
```
Typo here is what's causing the 404. 

Should be <tr>

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,301 @@
+            <td>Send messages to users in your app</td>
+        </tr>
+        <tr>
+            <td><strong>Surveys</strong></td>
```
We could technically do this - would just be a case of building one with an FF. The NPS app I'm working on could count towards this, for example.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,301 @@
+            <td>Recruit and schedule customer interviews. </td>
+        </tr>
+        <tr>
+            <td><strong>Apps/Integrations</strong></td>
```
```suggestion
            <td><strong>Apps</strong></td>
```
Would prefer to avoid confusion and keep naming consistent

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,301 @@
+</table>
+</div>
+
+- **Session replays:** [Session replays](/session-replay) in PostHog recreate exactly what real users see and how they use your product. They also enable you to debug problems using built-in console logs and network performance.
```
We could mention here that we include DOM explorer, performance monitoring, etc.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,301 @@
+
+- **Session replays:** [Session replays](/session-replay) in PostHog recreate exactly what real users see and how they use your product. They also enable you to debug problems using built-in console logs and network performance.
+
+- **Feature flags:** PostHog includes [multivariate feature flags](/feature-flags) that support JSON payloads, In-app prompts. Teams can use feature flags to offer different features or UI choices to users, to trigger in-app messages, and more.
```
```suggestion
- **Feature flags:** PostHog includes [multivariate feature flags](/feature-flags) that support JSON payloads. Teams can use feature flags to offer different features or UI choices to users, to trigger in-app messages, and more.
```

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,301 @@
+- **Feature flags:** PostHog includes [multivariate feature flags](/feature-flags) that support JSON payloads, In-app prompts. Teams can use feature flags to offer different features or UI choices to users, to trigger in-app messages, and more.
+
+- **A/B testing:** In PostHog, you can use the [experimentation suite](/ab-testing) to create multivariate tests within your product, such as showing some users a different page layout to others. Over time, you can build an understanding of which page performs better, correlate results with other events, and deploy a final version.
+
```
We could use the Tutorial Tracks component here to add some visual elements and further links to the page, as in the Pendo article: 
```
<h4 className="mb-0">Discover what's possible with <span className="text-blue">product analytics</span></h4>
<TutorialsSlider slugs={[
  "/tutorials/churn-rate",
  "/tutorials/funnels",
  "/tutorials/api-get-insights-persons",
  "/tutorials/feature-retention",
  "/tutorials/next-steps-after-installing"
]} />
```

joethreepwood reviewed (changes requested):
Some feedback and a cause for the 404. 

It feels a little thin at the moment, as we don't cover some of the core product features in much depth. Given Hotjar's focus on in-app messages it would be worth including information (and maybe screenshots, tutorial links) on how PostHog enables those features through FFs and build-your-own apps. 

Heatmaps are an area of weakness for us, as ours don't offer scrollmaps and the like - but it would be a fair comparison to include this. 

We also don't cover apps or interlinking of replays with other features here. The primary benefit we offer over Hotjar is that you can jump from analytics to replays to FFs, etc., whereas HJ keeps you limited to just one.

joethreepwood reviewed (changes requested):
Some changes, and a cause for the 404. 

It feels a bit thin at the moment. We don't cover heatmaps (which are an area of weakness for us, but a core feature for HJ) in depth at all, nor do we cover apps/integrations in much depth. This extra functionality -- along with offering a way to go from an insight to a replay in a single click -- is a key advantage of PostHog over Hotjar, so it feels worth bringing out. 

Personally, I'd also add more mention of in-app notifications and surveys as a thing which PostHog can do through FFs. We could include some screenshots and tutorial links which illustrate this.

andyvan-ph reviewed (commented):
+1 on Joe's comments atm, so won't go any deeper for now.

Lior539 commented on contents/blog/posthog-vs-pendo.md:
```diff
@@ -216,7 +216,7 @@ This difference is reflected in all levels of the product, but especially in pro
 - **SQL access:** While both PostHog and Pendo have ready-made insights and visualization types, only PostHog gives you unlimited access to your data by [writing your own SQL queries](/docs/product-analytics/hogql). This is ideal for data scientists, product managers, and engineers who want to perform advanced analysis on user data.
 
-<h4 className="mb-0">Discover what's possible with <span className="text-blue">product analytics</span></h4>
+<h4 className="mb-4">Discover what's possible with <span className="text-blue">product analytics</span></h4>
```
Before and after:

<img width="629" alt="Screenshot 2023-05-10 at 1 46 55 PM" src="https://github.com/PostHog/posthog.com/assets/7090054/a6cae059-7bec-4222-a2e7-e002f336f64e">
<img width="834" alt="Screenshot 2023-05-10 at 6 00 36 PM" src="https://github.com/PostHog/posthog.com/assets/7090054/fcfa7a8a-ba08-4c15-851c-b4f358704846">

joethreepwood commented:
Quick skim and looks a lot better. Two nits though. 

<img width="724" alt="Screenshot 2023-05-11 at 09 38 20" src="https://github.com/PostHog/posthog.com/assets/84011561/fe9c5b0b-4eb1-41e1-a8a2-fc04522da6b1">

Probably worth changing the colour of the text here, given that this orange colour is also how we stylize links and these aren't that. We probably want the text size to be consistent across the page too. 

<img width="1697" alt="Screenshot 2023-05-11 at 09 39 05" src="https://github.com/PostHog/posthog.com/assets/84011561/bc55fdb6-7147-474b-9ad0-4622b3de4ae9">

You've got a scroll bar on the bottom of the page. Not sure why.

ivanagas commented:
> You've got a scroll bar on the bottom of the page. Not sure why.

It is [this bug.](https://github.com/PostHog/posthog.com/issues/5857)

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+
+### 2. PostHog is built for engineers
+
+As such, PostHog includes many powerful features that aren’t available in tools like Hotjar, which is built for more general audiences. These features include tools like feature flags with JSON payloads, which enable you to trigger in-app messages or other notifications. 
```
```suggestion
As such, PostHog integrates numerous powerful features that aren’t available in tools like Hotjar, such as feature flags with JSON payloads, which enable you to trigger in-app messages, or other notifications. 
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+
+As such, PostHog includes many powerful features that aren’t available in tools like Hotjar, which is built for more general audiences. These features include tools like feature flags with JSON payloads, which enable you to trigger in-app messages or other notifications. 
+
+We built PostHog to support technically-savvy product managers and engineers — especially [engineers with a product focus in their role](/blog/what-is-a-product-engineer). Although anyone can use PostHog ([just ask our marketing team](blog/posthog-marketing)), our goal is to help engineers be better at product by giving them the insights and tools they need to ship impactful features at speed.
```
```suggestion
We built PostHog to support technically-savvy product managers and engineers – especially [engineers with a product focus in their role](/blog/what-is-a-product-engineer). Anyone can use PostHog ([just ask our marketing team](blog/posthog-marketing)), but our primary goal is to give engineers the tools they need to ship impactful features at speed.
```

andyvan-ph replied:
```suggestion
We built PostHog to support technically-savvy product managers and engineers – especially [engineers with a product focus in their role](/blog/what-is-a-product-engineer). Anyone can use PostHog ([just ask our marketing team](blog/posthog-marketing)), but our primary goal is to give engineers the tools they need to ship impactful features at speed.
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+    <tbody>
+        <tr>
+            <td><strong>Free recording allowance</strong></td>
+            <td className="text-center">~1,000 per month (max 35 per day) on Basic plan</td>
```
```suggestion
            <td className="text-center">~1k per month (max 35 per day) on Basic plan</td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+        <tr>
+            <td><strong>Free recording allowance</strong></td>
+            <td className="text-center">~1,000 per month (max 35 per day) on Basic plan</td>
+            <td className="text-center">15,000 per month</td>
```
```suggestion
            <td className="text-center">15k per month</td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+            <td>Free recordings before being charged</td>
+        </tr>
+        <tr>
+            <td><strong>Price for first 50,000 sessions</strong></td>
```
```suggestion
            <td><strong>Price for first 50k sessions</strong></td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+        </tr>
+        <tr>
+            <td><strong>Price for first 50,000 sessions</strong></td>
+            <td className="text-center">~$200 per month (max 1,500 sessions per day)</td>
```
```suggestion
            <td className="text-center">~$200 per month (max 1.5k sessions per day)</td>
```

andyvan-ph replied:
```suggestion
            <td className="text-center">~$200 per month (max 1.5k sessions per day)</td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+            <td></td>
+        </tr>
+        <tr>
+            <td><strong>Price for first 50,000 sessions</strong></td>
```
```suggestion
            <td><strong>Price for first 50k sessions</strong></td>
```

andyvan-ph replied:
```suggestion
            <td><strong>Price for first 50k sessions</strong></td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+        </tr>
+        <tr>
+            <td><strong>Price for first 50,000 sessions</strong></td>
+            <td className="text-center">~$200 per month (max 1,500 sessions per day)</td>
```
```suggestion
            <td className="text-center">~$200 per month (max 1.5k sessions per day)</td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+            <td><strong>Free recording allowance</strong></td>
+            <td className="text-center">~1,000 per month (max 35 per day) on Basic plan</td>
+            <td className="text-center">15,000 per month</td>
+            <td>Free recordings before being charged</td>
```
```suggestion
            <td></td>
```

andyvan-ph replied:
Don't think this needs explaining

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+        </tr>
+        <tr>
+            <td><strong>Connected to product analytics</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span> Requires additional integration</td>
```
```suggestion
            <td className="text-center">Requires additional integration</td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+        </tr>
+        <tr>
+            <td><strong>Console logs</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span> Only in Scale plan</td>
```
```suggestion
            <td className="text-center">Only in Scale plan</td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+
+### In-app messages and user research
+
+Hotjar enables you to hear from users with feedback widgets, surveys, and an integrated user research tool. PostHog also supports [feedback widgets](/apps/feedback-widget) and [user research scheduling](/apps/user-interviews), although not surveys. However, PostHog offers additional ways to communicate with your users, such as a [notification bar](/apps/notification-bar) and interactive [pop-ups] ...
```
```suggestion
Hotjar includes feedback widgets, surveys, and an integrated user research tool for gathering feedback from users. PostHog also supports [feedback widgets](/apps/feedback-widget) and [user research scheduling](/apps/user-interviews), although not surveys.

PostHog also offers additional ways to communicate with your users, such as a [notification bar](/apps/notification-bar) and interactive [pop-ups](tutorials/react-popups) through the use of [feature flag payloads](/docs/feature-flags/payloads), and [site apps](/tutorials/build-site-app). Also, since PostHog provides product analytics, you can easily define your target users who should see your widgets.
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+        </tr>
+        <tr>
+            <td><strong>Custom user targeting</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span> Requires additional integration</td>
```
```suggestion
            <td className="text-center">Requires additional integration</td>
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+
+One unique advantage of PostHog is that, because it is open source, it’s easy to [create your own apps and integrations](/tutorials/build-your-own-posthog-app). This is useful if you’re, for example, using niche software in your stack that hasn’t been widely adopted, or if you require a direct integration between PostHog and your product. 
+
+> The number of available apps is constantly increasing for both PostHog and Hotjar, so rather than list all available apps, we’ve shortened this section to only list the most popular integrations in particular categories. Want the full list? Check [the PostHog app library](/apps)!
```
```suggestion
```

Feels repetitive given we have a boxout after the table as well.

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+</table>
+</div>
+
+> **Want more?** For a full list of PostHog’s available integrations, please [check the app directory](/apps).
```
```suggestion
> **Want more?** Visit the [PostHog the app directory](/apps) for a full list of supported integrations.
```

andyvan-ph commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,537 @@
+
+Hotjar has tier-based pricing: Basic, Plus, Business and Scale. All plans offer session recordings, although the costs and limits differ. Additional features such as API access and SAML single sign-on are only available on some plans.
+
+The free can only record up to 35 daily session recordings. After this, the Plus tier offers 100 daily session recordings for $32 per month and the Business tier offers 500 daily sessions for $80 per month (with the option to pay for more sessions). Finally, the Scale tier offers 500 daily sessions for $171 per month, but offers access to additional features such as Console tracking and the Hotja ...
```
```suggestion
Hotjar's free plan only records up to 35 daily session recordings. After this, the Plus tier offers 100 daily session recordings for $32 per month and the Business tier offers 500 daily sessions for $80 per month (with the option to pay for more sessions). Finally, the Scale tier offers 500 daily sessions for $171 per month, but offers access to additional features such as Console tracking and the Hotjar API.
```

andyvan-ph reviewed (commented):
I've left a bunch of suggestions, but it's mostly light edits and formatting bits. 

Let me know if you have questions about edits, but otherwise this is pretty close to ready.

andyvan-ph commented:
@joethreepwood Would you mind giving this a final pass just to check we haven't missed anything important?

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+In this guide, we'll cover:
+
+- Hotjar and PostHog [feature comparison](#core-features)
+- [Session replay](#session-replay) capabilities
```
```suggestion
- [Session replay](#session-replays) capabilities
```
broken link, fixed

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+
+### 2. PostHog is built for engineers
+
+As such, PostHog integrates numerous powerful features that aren’t available in tools like Hotjar, such as feature flags with JSON payloads, which enable you to trigger in-app messages, or other notifications. 
```
```suggestion
PostHog integrates powerful features that aren’t available in Hotjar and which are designed for technical users, such as feature flags with JSON payloads, which enable you to trigger in-app messages, or other notifications. 
```

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+            <td><strong>Session replays</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Watch real users use your product; diagnose bugs</td>
```
```suggestion
            <td>Watch real users use your product</td>
```
Not it's only use

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+            <td>In-app widget for requesting customer interviews.</td>
+        </tr>
+        <tr>
+            <td><strong>User research tooling</strong></td>
```
I don't know what we're referring to here, or how it's different to the interview scheduling, etc.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+            <td></td>
+        </tr>
+        <tr>
+            <td><strong>Price for first 50k sessions</strong></td>
```
This is a repeat of the previous line.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+        </tr>
+        <tr>
+            <td><strong>Connected to product analytics</strong></td>
+            <td className="text-center">Requires additional integration</td>
```
I'd just change this to a no. Hotjar doesn't do this; it requires an additional tool that does it.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+            <td><strong>Console logs</strong></td>
+            <td className="text-center">Only in Scale plan</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Debug user problems and app errors.</td>
```
Nit: I won't correct this throughout, but this article is inconsistent in whether it uses a fullstop at the end of a sentence in this column. I don't mind having one, or not - but it should be a consistent style throughout the article.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+            <td>Debug user problems and app errors.</td>
+        </tr>
+        <tr>
+            <td><strong>Network activity</strong></td>
```
```suggestion
            <td><strong>Performance monitoring</strong></td>
```

joethreepwood replied:
```suggestion
            <td><strong>Performance monitoring</strong></td>
```

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+</table>
+</div>
+
+- **Recording retention:** Hotjar keeps all recordings for 365 days. PostHog retains all recordings for three weeks, but retains recordings saved to a playlist forever. PostHog also allows you to download recordings as a `.json` file for safe keeping.
```
```suggestion
- **Recording retention:** Hotjar keeps all recordings for 365 days. PostHog retains all recordings for three weeks, but retains recordings saved to a playlist forever. PostHog also enables you to download recordings as a `.json` file for safe keeping.
```

Tiny tone thing. Allows sounds permissive. Enable sounds empowering. I always prefer enable.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+### In-app messages and user research
+
+Hotjar includes feedback widgets, surveys, and an integrated user research tool for gathering feedback from users. PostHog also supports [feedback widgets](/apps/feedback-widget) and [user research scheduling](/apps/user-interviews), although not surveys.
+
```
```suggestion

However, because PostHog is entirely open source, it's possible for users to build their own user surveys if needed. 

```

You may want to expand on this, but I think it's good to add. Could arguably be a callout with a `>`

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+			<td className="w-3/12"></td>
```
```suggestion
	<td className="w-3/12"></td>
```
tidying up

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+        </tr>
+        <tr>
+            <td><strong>Custom user targeting</strong></td>
+            <td className="text-center">Requires additional integration</td>
```
Again, if the tool doesn't do it then I'd just say no here.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+            <td>Capture user feedback with surveys. </td>
+        </tr>
+        <tr>
+            <td><strong>User research tooling</strong></td>
```
Again, this is really vague. We should be specific about what this means, as the current description doesn't tell me anything.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+</div>
+
+### Apps and integrations
+Both Hotjar and PostHog have a wide selection of apps and integration. Both also include integrations with tools such as Zapier, which enable you to move data to even more platforms.
```
```suggestion
Both Hotjar and PostHog have a wide selection of apps and integrations. Both also include integrations with tools such as Zapier, which enable you to move data to even more platforms.
```

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+### Apps and integrations
+Both Hotjar and PostHog have a wide selection of apps and integration. Both also include integrations with tools such as Zapier, which enable you to move data to even more platforms.
+
+One unique advantage of PostHog is that, because it is open source, it’s easy to [create your own apps and integrations](/tutorials/build-your-own-posthog-app). This is useful if you’re, for example, using niche software in your stack that hasn’t been widely adopted, or if you require a direct integration between PostHog and your product. 
```
Love this

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+One unique advantage of PostHog is that, because it is open source, it’s easy to [create your own apps and integrations](/tutorials/build-your-own-posthog-app). This is useful if you’re, for example, using niche software in your stack that hasn’t been widely adopted, or if you require a direct integration between PostHog and your product. 
+
+
+Below, we've listed a few of the most popular integrations used across PostHog and Pendo. 
```
```suggestion
Below, we've listed a few of the most popular integrations used across PostHog and HotJar. PostHog offers more than 50 apps, so for a full list of PostHog's integrations, please [check the app library](/apps).
```
HotJar, not Pendo. Think this may have been copied from an earlier article? 

Also throwing in a CTA to the apps page

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+    <tbody>
+        <tr>
+            <td><strong>Hubspot</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span> Surveys only</td>
```
I'd just say no here, and maybe move the 'Surveys only' point to the notes column. It's confusing as-is, because we're defining a Hubspot app as a way to sync data, saying HotJar doesn't have one, but it does have one, just one that doesn't fit that definition.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+        <tr>
+            <td><strong>HIPAA compliant</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center">Self-host only</td>
```
We don't offer self-hosting anymore, other than via Open Source deployments, which wouldn't be suitable for a BAA.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+            <td>Remove personally identifiable information</td>
+        </tr>
+        <tr>
+            <td><strong>Self-hosting option</strong></td>
```
This needs to either be clear that we're talking about the open source Docker Compose version, or it needs to be corrected. We don't offer or support self-hosted versions as we used to.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Self-host and keep complete control of your data</td>
+        </tr>
```
We could add a point about cookie-less tracking here. Potentially also mention apps such as the Filter-Out app, which prevent unwanted data being ingested.

Lior539 replied:
Re cookie-less tracking: It breaks a whole bunch of session replay functionality, so I dont think it's worth including 
https://posthog.com/tutorials/cookieless-tracking

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+
+## Frequently asked questions
+
+- [How is PostHog different?](#how-is-posthog-different)
```
We don't need to repeat many of these headings at the end of the article? We can cut this down to just the ones which are actually in the FAQ section.

joethreepwood replied:
Not sure what you mean here about it being auto-populated or uneditable -- and the right hand side will pull in all headings regardless of them being repeated here.

I've committed in the suggested change so that we don't repeat the already-used headings needlessly at the end of the article and only show the questions which are actually asked in this section.

joethreepwood replied:
Relevant commit for comparison: https://github.com/PostHog/posthog.com/pull/5931/commits/09ab842bd70338b98539235b9a5387bfceede748

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+
+### Who is PostHog useful for?
+
+PostHog is built primarily with engineers, product managers and other technical users in mind. It also offers the ability to record session replays, heatmaps, and create feedback widgets, but also offers a number of other tools that are useful for these types of teams, such as product analytics and feature flags. 
```
Generally, I think we should be throwing links to relevant docs and product pages in here.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+
+### How much does Hotjar cost?
+
+Hotjar has tier-based pricing: Basic, Plus, Business and Scale. All plans offer session recordings, although the costs and limits differ. Additional features such as API access and SAML single sign-on are only available on some plans.
```
```suggestion
Hotjar has tier-based pricing: Basic, Plus, Business and Scale. All plans offer session recordings, although the costs and limits differ. Additional features such as API access and SAML single sign-on are only available on some plans.
```

Best to be specific about which plans these features are on, or how much they cost.

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,538 @@
+
+PostHog has transparent pricing based on the usage. It’s free to get started and completely free for the first 15,000 sessions and 1 million events captured every month.
+
+After this free monthly allowance you'll pay $0.005/recording and $0.00045/event, and PostHog charges progressively less the more you use. Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing.
```
```suggestion
After this free monthly allowance you'll pay from $0.005/recording and $0.00045/event, with charges decreasing the more you use. Volume, non-profit and [startup discounts](/startups) are available upon request, and we recommend trying [our pricing calculator](/pricing) to estimate your pricing.
```

joethreepwood reviewed (commented):
Left a bunch of comments and corrections. 

I think it's worth bringing product analytics into this. Hotjar _does_ offer product analytics features (funnels and trends), despite this saying they don't. Those tools are a lot more limited than PostHog's obviously, so it's something we can explore. 

We could also throw in a link to a few case studies which mention session replays, but that's not an essential inclusion. 

Pricing-wise, it's also worth pointing out that Hotjar gates many of its features behind different packages: Observe, Engage and Ask -- each of which has its own tiers. The pricing is so complex that it's worth explaining exactly how it works and that you have to pay for each package separately to get comparable features to PostHog's transparent, simple, all-in-one cost.

Lior539 commented:
Thanks for the feedbak @joethreepwood. I think it's looking better now:

- Added product analytics comparison
- Added deep dive into Hotjar pricing (geez its complicated)
- Added Contra case study

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,1020 @@
+            <td><strong>User interview recordings and transcriptions</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td></td>
```
Missing a note or description here (and in all subsequent mentions of it)

Lior539 replied:
Hmm Im not sure this needs a note

joethreepwood commented on contents/blog/posthog-vs-hotjar.md:
```diff
@@ -0,0 +1,1020 @@
+
+Session replays are an essential tool for understanding how people use your product, especially for [early-stage companies](/blog/early-stage-analytics) searching for product-market fit. 
+
+> Learn [how Contra used session replays](/customers/contra) to increase registrations by 30%. 
```
```suggestion
> **Ready to learn more?** Find out [how Contra used PostHog's session replays](/customers/contra) to increase registrations by 30%. 
```
Was expecting us to work it into the copy, but I like this CTA. Will defer to @andyvan-ph on if it's too heavy-handed though, in which case we can just lose the `>`

joethreepwood reviewed (approved):
Love the new pricing table!
